app/enhancements/enhancement.rb

```diff
@@ -0,0 +1,17 @@
+##
+# Enhancements take a SolrProperty and return another SolrProperty with new
+# values.
+class Enhancement
+  attr_reader :raw_property
+
+  # @param [SolrProperty] raw_property The property to transform
```

Comment:
I wanted to add YARD to the base class. Technically this class isn't required at all - maybe I should ditch it?